src/AccessibilityInsights.SharedUx/Utilities/ContainedControlFinder.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Windows;
+
+namespace AccessibilityInsights.SharedUx.Utilities
```

Comment:
I pulled this out and used a template, just in case we need the functionality elsewhere. It's internal but could easily be public if we wanted it to be